src/tribler-core/tribler_core/utilities/osutils.py

```diff
@@ -235,3 +235,16 @@ def get_root_state_directory(home_dir_postfix=u'.Tribler'):
         path = os.environ['TSTATEDIR']
         return Path(path) if os.path.isabs(path) else Path(os.getcwd(), path)
     return Path(get_appstate_dir(), home_dir_postfix)
+
+
+def reset_env_state_directory(state_dir: str):
```

Comment:
This method is only used by the debug endpoint tests. Can you move it to `test_debug_endpoint.py`, instead of having it in this file?

Reply:
sure